sql-parser/src/main/java/io/crate/sql/tree/DefaultTraversalVisitor.java

```diff
@@ -325,9 +325,11 @@ public R visitAlterTable(AlterTable<?> node, C context) {
     }
 
     @Override
-    public R visitInsertFromSubquery(InsertFromSubquery<?> node, C context) {
+    public R visitInsert(Insert<?> node, C context) {
         node.table().accept(this, context);
-        node.subQuery().accept(this, context);
+        if (node.subQuery() != null) {
```

Comment:
there always should be an insert source.

Reply:
Great! thanks for the clarification, i wasn't sure about this. Will drop the null checks etc. then.